apps/console/src/auth.html

```diff
@@ -179,13 +180,18 @@
                 sessionStorage.setItem("auth_callback_url_console", authCallbackUrl);
             }
 
-            var serverOrigin = startupConfig.serverUrl;
+            if (isCentralDeploymentEnabled) {
+                localStorage.setItem("user_requested_tenant", userTenant);
```

Comment:
Hope we are clearing this on user logout.

Reply:
We clear this on this page itself after utilising this value